frameworks/cassandra/tests/conftest.py

```diff
@@ -1,10 +1,15 @@
 import pytest
+import sdk_login
 import sdk_repository
 import sdk_security
 
 
 @pytest.fixture(scope='session')
-def configure_universe():
+def configure_login():
```

Comment:
This invocation is identical in all of the framework test suites so I am fine if you make it an implicit (i.e. autouse) or explicit fixture at the top-level and thereby avoid having to make all these small changes.

One advantage of doing this at the top-level conftest.py with implicit injection is that you could cd down to this individual directory and login yourself and not deal with this new interface or simply avoid having the test reconfigure your local CLI

However, that's a pretty specific scenario and stuff at the top level might just need to be copied out anyway if we split these tests into their own repos. Furthermore the function you wrote already has a clear override mechanism so its not strictly necessary.